docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1358,6 +1358,10 @@ Sets a global limit on number of users that can be logged in at one time. Defaul
 
 Sets a global limit on number of alert rules that can be created. Default is -1 (unlimited).
 
+### global_correlations
+
+Sets a global limit on number of correlations that can be created. Default is -1 (unlimited).
```

Comment:
Typo: should be 'the number'. 

Also, 'correlations that can be created' is passive voice. Who is creating the correlations? I assume the user?

If it's the users, I'd say "Sets a global limit on the number of correlations you can create. The default value is `-1`, allowing unlimited correlations."

Reply:
@chri2547 I followed what I thought was standard based on other quotas documented.

The user does create correlations. 

![Screenshot 2023-03-22 at 11 58 29 AM](https://user-images.githubusercontent.com/1533128/226981188-e90582d1-a7d7-4479-b1d0-665f32fa8972.png)
![Screenshot 2023-03-22 at 11 59 26 AM](https://user-images.githubusercontent.com/1533128/226981191-2ca7251d-c8f8-4ac8-9df0-5ff6c49ed562.png)

I can fix my quota, but I'm thinking reworking this file to fit the standards you're applying here will be a better path forward.

Reply:
@gelicia , You are correct that the file should be updated accordingly. I did introduce inconsistency, sorry about that. I think the words you used are fine as is, and if there's time we can work to update all the text.